client/Cargo.toml

```diff
@@ -49,4 +52,3 @@ features = [
   'WebGlTexture',
   'WebGlUniformLocation',
 ]
-
```

Comment:
something something [POSIX newline at the end of the file](https://unix.stackexchange.com/a/18789)